net/nimble/controller/src/ble_ll_dtm.c

```diff
@@ -315,6 +315,10 @@ ble_ll_dtm_rx_start(void)
 
     ble_ll_state_set(BLE_LL_STATE_DTM);
 
+#ifdef BLE_XCVR_RFCLK
+    ble_ll_xcvr_rfclk_timer_start(os_cputime_get32());
```

Comment:
this will try to setup timer for starting clock which is pointless since you want to start it now.

if (ble_ll_xcvr_rfclk_state() == BLE_RFCLK_STATE_OFF) {
    ble_ll_xcvr_rfclk_start_now(os_cputime_get32());
}

Reply:
agree.